systemtest/src/test/java/io/strimzi/systemtest/upgrade/OlmUpgradeST.java

```diff
@@ -77,6 +78,8 @@ void testStrimziUpgrade(ExtensionContext extensionContext) throws IOException {
         procedures.put("logMessageVersion", testKafkaVersion.messageVersion());
         procedures.put("interBrokerProtocolVersion", testKafkaVersion.protocolVersion());
         latestUpgradeData.put("proceduresAfterOperatorUpgrade", procedures);
+        // This is needed to overwrite for 2.0.0 OLM
+        latestUpgradeData.put("fromVersion", "1.8.0");
```

Comment:
There is no 1.8.0 / 2.0.0 => should this be parametrized?

Reply:
That's just some leftovers, gonna remove it, thanks